home-assistant-plugin/custom_components/xknx/__init__.py

```diff
@@ -156,26 +154,23 @@ async def async_setup(hass, config):
     for platform_config, device_type in KNX_CONFIG_PLATFORM_MAPPING.items():
```

Comment:
what is KNX_CONFIG_PLATFORM_MAPPING for?
{"switch": "switch",
"light": "light",
...} 
platform_config and device_type seems to be equal.